pkgs/development/ocaml-modules/lablgtk3/default.nix

```diff
@@ -1,27 +1,38 @@
-{ stdenv, fetchurl, pkgconfig, ocaml, findlib, gtk3, gtkspell3, gtksourceview }:
+{ stdenv,lib, fetchFromGitHub, pkgconfig, ocaml, findlib, dune, gtk3, cairo2 }:
 
-if !stdenv.lib.versionAtLeast ocaml.version "4.05"
+if !lib.versionAtLeast ocaml.version "4.05"
 then throw "lablgtk3 is not available for OCaml ${ocaml.version}"
 else
 
+# This package uses the dune.configurator library
+# It thus needs said library to be compiled with this OCaml compiler
```

Comment:
If this is a common need, we should definitely consider writing a wrapper for this or, at least, document it properly in the OCaml section of the nixpkgs doc. But honestly, my preferred solution is still to just move `dune` to `ocamlPackages` and be done with it.